src/main/resources/application.yaml

```diff
@@ -439,6 +442,9 @@ notification:
         representative:
           emailId: 0d844af4-b390-42d7-94d5-4fd1ae9388d9
           smsId: 9fe3a4f1-b8e2-4aed-aafb-6360d6ba2874
+        jointParty:
```

Comment:
Shouldn't this by joint_party?

Reply:
yes it should

Reply:
I removed it, it didn't belong as it was for oral instead of paper.